src/utils/ParseRouteUtils.js

```diff
@@ -304,17 +359,35 @@ function parseWalkingRoute(data: any, dateMs: number, destinationName: string, i
       startTime: departureTime,
       stops: [],
       tripIdentifiers: null,
-      type: 'walk',
+      type: DIRECTION_TYPE.WALK,
     };
 
+    const routeSummary = [
+      {
+        stopName: originName,
+        direction: { type: DIRECTION_TYPE.WALK },
+        stayOnBusForTransfer: false,
+      },
+      {
+        stopName: destinationName,
+        direction: null,
+        stayOnBusForTransfer: false,
+      },
+    ];
+
     return adjustRouteTimesIfNecessary({
       arrivalTime,
       boundingBox,
       departureTime,
       directions: [direction],
       endCoords,
+      endName: destinationName,
```

Comment:
why are these different tho

Reply:
its what was requested by iOS but you're right, we can prob just change it

Reply:
actually wait no its b/c we have `startCoords` and `endCoords` so its more consistent to have `startName` and `endName`